edx_plugin/views.py

```diff
@@ -0,0 +1,6 @@
+from django.http import HttpResponse
+
+
+# test response
+def get_plugin_page(request):
+    return HttpResponse("This is plugin page")
```

Comment:
Make your second nature the usage of gettext on those strings which potentially may be faced by users.